### PR TITLE
Correct startup path

### PR DESCRIPTION
**Pull request by posix4e**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,9 +19,6 @@ script:
   - export PATH=/snap/bin:$PATH
   - sudo snapcraft cleanbuild
   - sudo cp *.snap "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
-after_success:
-  - sudo snap install transfer
-  - timeout 180 sudo /snap/bin/transfer "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
```
Comment (lucyllewy) [inline .travis.yml]: why did you delete this?

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -40,15 +40,14 @@ parts:
 
 apps:
   brave:
-    command: bin/desktop-launch $SNAP/usr/bin/brave --no-sandbox
+    command: bin/desktop-launch $SNAP/opt/brave.com/brave/brave  --no-sandbox
```
Comment (lucyllewy) [inline snap/snapcraft.yaml:47]: your previous PR removed the desktop file. Please add it back with the code that massages the Icon path.

File: .travis.yml (hunk 1 above)
Comment (posix4e) [reply .travis.yml]: Transfer.com is shutting down

File: snap/snapcraft.yaml (hunk 2 above)
Comment (posix4e) [reply snap/snapcraft.yaml:47]: Added back!

**Comment by posix4e** [comment]
Still working on it.

File: snap/snapcraft.yaml (hunk 2 above)
Comment (lucyllewy) [reply snap/snapcraft.yaml:47]: thanks for that. We also need to tell snapcraft where the desktop file is:

```suggestion
    command: bin/desktop-launch $SNAP/opt/brave.com/brave/brave  --no-sandbox
    desktop: usr/share/applications/brave-browser.desktop
```

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -15,11 +15,11 @@ architectures:
 parts:
   brave:
     plugin: dump
-    source: https://github.com/brave/brave-browser/releases/download/v0.55.22/brave-browser_0.55.22_amd64.deb
+    source: https://github.com/brave/brave-browser/releases/download/v0.56.12/brave-browser_0.56.12_amd64.deb
     source-type: deb
     # Correct path to icon.
     override-pull: |
-      snapcraftctl pull  && rm -rf etc/cron.daily/ &&  rm -rf usr/bin/brave-browser
+      snapcraftctl pull  && rm -rf etc/cron.daily/ &&  rm -rf usr/bin/brave-browser && chmod 755 opt/brave.com/brave/brave-sandbox && sed -i 's|Icon=brave-browser|Icon=/opt/brave.com/brave/product_logo_128\.png|g' usr/share/applications/brave-browser.desktop
```
Comment (lucyllewy) [inline snap/snapcraft.yaml]: would it be possible to convert this to multiple lines, please?

```suggestion
      snapcraftctl pull 
      rm -rf etc/cron.daily/
      rm -rf usr/bin/brave-browser
      chmod 755 opt/brave.com/brave/brave-sandbox
      sed -i 's|Icon=brave-browser|Icon=/opt/brave.com/brave/product_logo_128\.png|g' usr/share/applications/brave-browser.desktop
```

File: .travis.yml (hunk 1 above)
Comment (lucyllewy) [reply .travis.yml]: oh, is it? hmm @popey what should we do about that?

Comment (lucyllewy) [reply .travis.yml]: the transfer snap uses transfer.sh. Is that different to transfer.com or did you typo? Can you point me to the announcement that transfer.sh is shutting down because I can't seem to find any reference (I'm sure I'm failing at google. my fault, not yours :-) )

Comment (lucyllewy) [reply .travis.yml]: aha. there's a message on their homepage:

> Don’t Panic! transfer.sh will live on!
> After running and supporting transfer.sh for 4 years on my own, I’m happy to announce we are partnering with Storj Labs to keep the project going. From day one, the transfer.sh code has been open source. Storj has a commitment to open source sustainability and reached out to help us find a way to keep our project alive. Stay tuned for updates on the partnership with Storj, but for now, please continue to enjoy the service!
> Blue skies, Remco Verhoef


Looks like we're ok to continue using it :-)

Comment (posix4e) [reply .travis.yml]: @diddledan wanna add a patch to add it back? I don't use it

**Comment by posix4e** [comment]
BY the way this is now on edge

File: .travis.yml (hunk 1 above)
Comment (lucyllewy) [reply .travis.yml]: you might not, but we do to test pull requests before merging.
